### ISSUE TITLE
Problem with trying to use a path variable in a SOAP to REST conversion API

### ISSUE THREAD

**Issue by dumindurox777** [wso2/product-apim#7584]
### Description:
 Unable to capture the path parameter of a GET request in the inflow section. However,  unable to get this path parameter using a property mediator. 

### Steps to reproduce:
Adding a property mediator (inside a log mediator) in Publisher -> <API> -> Implement tab -> In Flow section as follows.

```
<log level="custom">
                <property xmlns:soapenv="http://www.w3.org/2003/05/soap-envelope"
                xmlns:ns3="http://org.apache.synapse/xsd"
                name="client"
                expression="get-property('uri.var.client')"/>
</log>
```
Path parameter is defined as the client in the API Design tab. However, we have observed that this change was not reflected in the synapse artifact which resides in the <APIM_HOME>/repository/deployment/server/synapse-configs/default/api directory after publishing the API.

### Affected Product Version:
APIM-2.6.0

### Optional Fields
https://github.com/wso2/carbon-apimgt/pull/8138/files

#### Suggested Labels:
apim-2.6.0

**Comment by npamudika** [comment]
Fixed with https://github.com/wso2/carbon-apimgt/pull/8138/

**Comment by npamudika** [comment]
Fixed in APIM 2.6 with https://github.com/wso2-support/carbon-apimgt/commit/8ce48d8c012df6b4edbb63716c54df43a70be3e9
